### PR TITLE
[Task 1849756] refactor MySQL connector UI components to share them with SQL server

### PR DESCRIPTION
**Pull request by shenqianjin**
[AB#1849756](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1849756)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java [hunk 1]
```diff
@@ -20,8 +21,9 @@
 import java.sql.Statement;
 import java.util.Collections;
 
-public class MySQLConnectionUtils {
+public class DatabaseConnectionUtils {
 
+    private static final String SQL_URL_PREFIX = "jdbc:sqlserver:";
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java:26]: rename to `SQLSERVER_URL_PREFIX`?

**Review by wangmingliang-ms — APPROVED** [review]

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java (hunk 1 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java]: renamed.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java [hunk 2]
```diff
@@ -31,7 +34,11 @@
 
     public static boolean connect(JdbcUrl url, String username, String password) {
         try {
-            Class.forName("com.mysql.jdbc.Driver");
+            if (StringUtils.startsWith(url.toString(), SQL_SERVER_URL_PREFIX)) {
+                Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
+            } else {
+                Class.forName("com.mysql.jdbc.Driver");
+            }
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java:41]:
```
Class.forName(getDriver(url));
...
```

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseConnectionUtils.java]: updated.

**Review by wangmingliang-ms — APPROVED** [review]

**Pull request merged**